src/elements/ngm-left-side-bar.js

```diff
@@ -36,6 +36,7 @@ class LeftSideBar extends I18nMixin(LitElement) {
       hideWelcome: {type: Boolean},
       mapChooser: {type: Function},
       authenticated: {type: Boolean},
+      globeQueueLength_: {type: Number},
```

Comment:
instead of having `globeQueueLength_` here it could be a "standard" variable and `this.requestUpdate()` called when the value is changed

Reply:
It is is a difficult decision:
- on an API point of view, we should not expose private properties to the outside;
- on a clarity point of view it makes more sense to let lit-element trigger renders (and also clearly define what are the dependencies of the template).

I would stay with the 2nd option. And maybe we can discuss this with the lit-element people, there are currently thinking about lit-element 3.x.